### ISSUE TITLE
auth() with import causes linking-error

### ISSUE THREAD

**Issue by platon-ivanov** [zenstackhq/zenstack#1257]
**Description and expected behavior**
`auth()` function fields aren't resolved when `User` model is imported.  

**Screenshots**
![image](https://github.com/zenstackhq/zenstack/assets/100810261/1eb2573f-ca06-489c-981e-11bfb71808ad)

**Environment (please complete the following information):**

-   ZenStack version: next
-   Prisma version: 5.5.2
-   Database type: Postgresql

**Additional context**

user.zmodel:

```prisma
import "./base"
import "./image"

model User extends Base {
    email String @unique

    @@auth
}
```

image.zmodel:
```prisma
import "./user"
import "./base"

model Image extends Base {
    width Int @default(0)
    height Int @default(0)

    @@allow('read', true)
    @@allow('all', auth().role == Admin)
}
```

**Comment by ymc9** [comment]
Thanks for reporting it @platon-ivanov ! Will make a fix and include in the upcoming new beta.

**Comment by ymc9** [comment]
Fixed in 2.0.0-beta.13
